paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentOptionsViewModel.kt

```diff
@@ -155,6 +155,9 @@ internal class PaymentOptionsViewModel @Inject constructor(
             }
         }
 
+        // This is bad, but I don't think there's a better option
+        PaymentSheet.FlowController.linkHandler = linkHandler
```

Comment:
Where do we clear this?

Reply:
Will do that based on the `FlowController` host’s lifecycle.